packages/core/src/utilities.ts

```diff
@@ -63,3 +64,10 @@ export function applyStyles(
 
   return flattenedStyles;
 }
+
+export function getRealValue(value: string | number | undefined) {
+  if (isString(value)) return value;
+  else if (isNumber(value)) return String(value);
+  else if (!value) return undefined;
```

Comment:
whats the purpose of undefined? is that bc nothing is selectedin the radio button, which makes it undefined?

does null work here too?

Reply:
undefines were being used as conditionals for value changes